test/avocado/run-tests

```diff
@@ -166,19 +166,21 @@ def run_avocado(avocado_tests, verbose, browser, download_logs):
     selenium = testvm.VirtMachine(image="selenium", label="selenium", verbose=verbose) if use_selenium else None
 
     try:
-        machine.start()
+        # just start the machine without waiting for it to boot, get the ip address later
+        machine.start(wait_for_ip=False)
         if selenium:
+            # actually wait here, because starting selenium takes a while
             selenium.start()
-
-        machine.wait_boot()
-        machine.start_cockpit()
-        if selenium:
             selenium.wait_boot()
 
             # start selenium on the server
             selenium.upload([ "avocado/selenium_start.sh"], "/root")
             selenium.execute(command="/root/selenium_start.sh")
 
+        machine.wait_for_ip_address()
+        machine.wait_boot()
```

Comment:
> Is there a reason not to include wait_for_ip_address() in wait_boot()?

I thought there was, but now after my refactoring I think I can put that in there.